docs/design/storagenode-downtime-tracking.md

```diff
@@ -155,67 +153,61 @@ For each node performs an _uptime check_.
 
-The services which update the `last_contact_failure` choose Storage Nodes randomly, hence we believe that these corner cases are minimal and losing some offline seconds tracking is acceptable and desirable for having a simpler solution.
+The services, which update the `last_contact_failure` choose storage nodes randomly, hence we believe that these corner cases are minimal and losing some offline seconds tracking is acceptable and desirable for having a simpler solution.
 
 Next, we present some alternative architectural solutions.
 
-### Independent Service
+### Independent Process
 
-The designed system runs in the Satellite single operative system process spinning up them in Goroutines. Currently, this is how all the different Satellite application processes run.
+Currently all chores and services run within a single process. Alternatively there could be an independent process for _offline downtime tracking_ as described in the [design section](#design).
 
-An alternative design is to create an independent service for running the process of the _offline downtime tracking_ described in the [design section](#design).
-
-The advantages of this alternative are:
+The advantages are:
 
-* It doesn't add a new application process to the Satellite.
+* It doesn't add a new application chore to the Satellite.
 * It's easier to scale.
 
 And the disadvantages are:
 
-* It requires to expose via a wire protocol the data selected from the nodes table. This cause more work and more latency apart from not offloading the current database<sup>1</sup>.
+* It requires to expose via a wire protocol the data selected from the nodes table. This adds more work and more latency apart from not offloading the current database<sup>1</sup>.
 * It requires to update the deployment process.
 
 The disadvantages outweigh the advantages of considering that:
 
-* We want to start to track Storage Nodes offline time
+* We want to start to track storage nodes offline time.
 * It doesn't offload the database despite being split in a different service.
-* It may clash with the team working in the distributed Satellite architecture, or it would require to synchronize the tasks, hence more work and time to release.
-
+* This approach conflicts with satellite separation, or it would require to synchronize the tasks.
 
 <sup>1</sup> We want to reduce calls to the current database.
 
 ### InfluxDB
 
-The designed system uses a SQL database for storing the Storage Nodes downtime.
-
-An alternative solution is to use [InfluxDB time-series database](https://www.influxdata.com/).
+The designed system uses a SQL database for storing the Storage Nodes downtime. Alternatively it could use [InfluxDB time-series database](https://www.influxdata.com/).
```

Comment:
```suggestion
The designed system uses a SQL database for storing the storage nodes downtime. Alternatively it could use [InfluxDB time-series database](https://www.influxdata.com/).
```